components/places/android/src/main/java/mozilla/appservices/places/PlacesConnection.kt

```diff
@@ -704,6 +722,24 @@ interface ReadableHistoryConnection : InterruptibleConnection {
      */
     fun getVisitPage(offset: Long, count: Long, excludeTypes: List<VisitType> = listOf()): List<VisitInfo>
 
+    /**
+     * Page more efficiently than using simple numeric offset. We first figure out
+     * a visited timestamp upper bound, then do a smaller numeric offset relative to
+     * the bound.
+     *
+     * @param offset The offset between first item that has visit date equal to bound
+     *  and last visited item.
+     * @param bound The upper bound of already visited items.
```

Comment:
As above, I think switching the order of `offset` and `bound` will make it easier to form a correct mental model of the query here.

Reply:
Done.